ukvm/ukvm-core.c

```diff
@@ -382,48 +381,6 @@ static void setup_cpuid(int kvm, int vcpufd)
         err(1, "KVM: ioctl (SET_CPUID2) failed");
 }
 
-#if 0
-static void inject_interrupt(int vcpufd, uint32_t intr)
-{
-    struct kvm_interrupt irq = { intr };
-
-    int ret = ioctl(vcpufd, KVM_INTERRUPT, &irq);
-
-    if (ret) {
-        printf("ret = %d\n", ret);
-        printf("errno = %d\n", errno);
-        err(1, "KVM_INTERRUPT");
-    }
-}
-#endif
-
-static pthread_cond_t sleep_cv = PTHREAD_COND_INITIALIZER;
-static pthread_mutex_t interrupt_mutex = PTHREAD_MUTEX_INITIALIZER;
-
-static void *event_loop(void *arg)
-{
-    struct kvm_run *run = (struct kvm_run *) arg;
-
-    /*
-     * We are using a TSC-based clock, but this is an example for
-     * delivering a timer interrupt once a second.
-     */
-    if (0) {
-        for (;;) {
-            sleep(1);		/* every 1 s */
-
-            pthread_mutex_lock(&interrupt_mutex);
-
-            run->request_interrupt_window = 1;
```

Comment:
This was the old primitive way of having a "tick" interrupt, definitely not needed anymore.